include/rocksdb/cloud/cloud_env_options.h

```diff
@@ -60,12 +60,48 @@ enum class CloudRequestOpType {
 using CloudRequestCallback =
     std::function<void(CloudRequestOpType, uint64_t, uint64_t, bool)>;
 
+class BucketOptions {
+private:
+  std::string bucket_;
+  std::string prefix_;
+  std::string object_;
+  std::string region_;
+  std::string name_;
+public:
+  BucketOptions();
+  // Sets the name of the bucket to be the new bucket name.
+  // If prefix is specified, the new bucket name will be [prefix][bucket]
+  // If no prefix is specified, the bucket name will use the existing prefix
+  void SetBucketName(const std::string & bucket,
+		     const std::string & prefix = "");
+  const std::string & GetBucketName() const { return name_; }
+  const std::string & GetObjectPath() const { return object_; }
+  void SetObjectPath(const std::string & object) { object_ = object; }
+  const std::string & GetRegion() const { return region_; }
+  void SetRegion(const std::string & region)  { region_ = region; }
+
+  // 
+  void TEST_Initialize(const std::string & name_prefix,
```

Comment:
Here and everywhere else: our code style doesn't put whitespace between `&` (or `*`) and the type name.

So: `const std::string& name_prefix`, *not* `const std::string & name_prefix`